gradle/libs.versions.toml

```diff
@@ -24,7 +24,7 @@
 ###
 
 [versions]
-com_amazonaws_aws_kms = "1.12.538"
+com_amazonaws_aws_kms = "1.12.668" # Addresses https://github.com/TBD54566975/web5-kt/security/advisories/GHSA-g6qj-pj38-2465
```

Comment:
```suggestion
com_amazonaws_aws_kms = "1.12.668"
```

Reply:
Why remove the doc to future maintainers about why this version, @frankhinek?

Reply:
i think it may be because we might change this version again and then the comment can easily get left "orphaned"?